tests/unit/type/zset.tcl

```diff
@@ -1697,7 +1712,10 @@ start_server {tags {"zset"}} {
             set original_max_value [lindex [r config get zset-max-ziplist-value] 1]
             r config set zset-max-ziplist-value 10
             create_zset myzset $contents
+            set card [r zcard myzset]
+
             assert_encoding $type myzset
+            assert_equal $card 10
```

Comment:
Why are you add this assert.
It looks no-related to this pr.

Reply:
i suppose it's related to the cleanup (use of `$card` below)

Reply:
I think it can be used in the following test cases, so as to avoid hard-coded 10 20.
indeed no-related...

Reply:
i don't have anything against it.. you can re-add if it you want.
the assertion is kinda trivial, so may not be needed..
maybe just call SCARD at the bottom part, before the code that uses it?
although truth be told.. this entire test is coupled with the fact there are exactly 10 items in the set (we call ZRANDMEMBER with count below 10, above 10, and exactly 10, in order to hit various edge cases)

Reply:
I actually revert it before you commented... 
I saw your comment as soon as I submitted it